blocks/how-to/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@blocks/how-to",
+  "version": "0.1.0",
+  "description": "Describe a task or process via a list of steps ",
+  "keywords": [
+    "blockprotocol",
+    "blocks"
+  ],
+  "license": "MIT",
+  "author": "luisbettencourt",
+  "scripts": {
+    "build": "block-scripts build",
+    "dev": "block-scripts dev",
+    "format": "prettier --write --ignore-unknown .",
+    "generate-types": "block-scripts codegen",
+    "lint:tsc": "tsc --noEmit",
+    "serve": "block-scripts serve"
+  },
+  "prettier": {
+    "printWidth": 80,
+    "trailingComma": "all"
+  },
+  "dependencies": {
+    "@blockprotocol/graph": "0.1.0-canary-20230227204452",
+    "@fortawesome/free-regular-svg-icons": "6.0.0",
+    "@fortawesome/free-solid-svg-icons": "6.0.0",
+    "@hashintel/design-system": "0.0.4",
+    "@mui/material": "5.11.8"
+  },
+  "devDependencies": {
+    "@types/react-dom": "18.0.8",
+    "block-scripts": "0.1.0-canary-20230227204452",
+    "mock-block-dock": "0.1.0-canary-20230227204452",
+    "prettier": "2.8.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "peerDependencies": {
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
+  },
+  "block-scripts": {
+    "port": 63212
+  },
+  "blockprotocol": {
+    "blockType": {
+      "entryPoint": "react"
+    },
+    "displayName": "How To",
+    "examples": [
+      {
+        "title": "Boil water",
+        "description": "Bring the clear water to a rolling boil for 1 minute"
+      }
+    ],
```

Comment:
I'm unsure if the "examples" fields is even used anymore, but if it is the spec current'y says it is a `properties` object, which means you are correct and it should be Base URLs.

Reply:
It is used in the Hub for the starting demo data

Reply:
Can you point me to the code where this happens? Do we create the `EntityMetadata` and such and wrap it?

Reply:
Metadata comes from the db, the block hub takes the first example from it (if it exists) https://github.com/blockprotocol/blockprotocol/blob/9b79035877eb371d7f0c82059b0ead6179ff6be8/apps/site/src/components/pages/hub/block-data-container.tsx#L72

Reply:
This sets the starting value for the JSON editor on the right of the block demo (that editor just being for the properties)

Reply:
Gotcha, we may want to look at updating this today / this week to be a whole entity, otherwise it will be impossible for someone to supply a valid link entity if for some reason they've made their block entity a link entity

Reply:
We can look at that – suggest not today

Reply:
Fixed the property names in 7f645a44768bcd6d7cee049f4ebcd3a9aa8b6bd8